app/controllers/categories_controller.rb

```diff
@@ -334,10 +353,7 @@ def search
         ) if term.present?
 
     categories =
-      categories.where(
-        "id = :id OR parent_category_id = :id",
-        id: parent_category_id,
-      ) if parent_category_id.present?
```

Comment:
What is the motivation for changing this?

Reply:
This was a bug. It returned the parent category too, when it should've returned only children categories.

Reply:
:+1: